### PR TITLE
fix overlapping text

### LINKED ISSUE #5: Disclaimer text overlaps over GitHub logo link
# Title says all :P
Example screenshots:
image1
image2
image3

## Ways to remedy this:
idk, maybe adjust the font with a media call (text might get unreadable though)
or adjusting the margin may be a good way to fix this

### PR DESCRIPTION
**Pull request by eveee00**
alright, it's kinda working, just snapping to the wrong side. anyways, I'm done for today and will fix this later. (@Icey67 or you can fix it too if you want.)

fixes #5

### CODE REVIEW

**Comment by Icey67** [comment]
> OHMYGAD IT WORKS



Cannot observe how it works since I also just woke up

**Comment by Icey67** [comment]
And with that you are free to merge it :shipit:

**Comment by eveee00** [comment]
wooooo

**Comment by eveee00** [comment]
goddamnit, the only issue this now has is that the text is snapping to the left.

**Comment by Icey67** [comment]
Screenshot_20240929-072259_Brave
Looks good tho 👍

**Comment by eveee00** [comment]
thanks!

**Comment by Icey67** [comment]
though I'd still recommend using this :P
Screenshot_20240929-072523_Brave

File: style.css [hunk 1]
```diff
@@ -119,33 +120,39 @@ body {
...
+        display: flex;
+        flex-direction: row;  /* Ensure text and logo are side by side */
+        justify-content: space-between;
+        align-items: center;
+        padding-right: 50px;  /* Ensure space for the logo */
+    }
 }
-
-
+/* is this even still necessary? -i67; remov ing the code didn't fuck anything up, so no.*/
```
Comment (eveee00) [inline style.css:155]: epic space

**Pull request merged**